Jenkinsfiles/signature-verification.sh

```diff
@@ -53,10 +53,18 @@ COPY="nix copy --from https://cache.vedenemo.dev"
 [[ ! -e "$path_to_check" ]] && $COPY "$path_to_check"
 [[ ! -f "$signature_file" ]] && $COPY "$signature_file"
 
+
 # get the sha256 of the file being verified
 PATH_HASH="$(Calc_sha256sum "$path_to_check")"
 
-# verify the file using given signature_file
-# start.sh verify returns zero if signature_file is valid.
-# Error code 10 for invalid signature_file, other codes signal that something else went wrong
-scs-pki-research/yubikey/start.sh verify "-h=$PATH_HASH" "-s=$(<"$signature_file")"
+# store hash to a file
+echo "$PATH_HASH" > digest.hex
+
+# convert the hashfile into binary format
+xxd -r -p digest.hex digest.bin
+
+# convert signature file to signature.bin file
+openssl enc -base64 -d -in "$signature_file" -out signature.bin
+
+# validate the authenticity of the signature
+openssl dgst -sha256 -verify ganymede.pem -signature signature.bin digest.bin
```

Comment:
"ganymede.pem" hardcoded, maybe this should be dug up from the signature file name.

Again see ci-private/web/srv/web/verify-signature.sh for reference